ruff_dev/src/main.rs

```diff
@@ -22,6 +23,8 @@ enum Commands {
     GenerateSourceCode(generate_source_code::Cli),
     /// Print the AST for a given Python file.
     PrintAST(print_ast::Cli),
+    /// Print the LibCST CST for a given Python file.
+    PrintCST(print_cst::Cli),
```

Comment:
I renamed this to `PrintCST`, since the CST in `LibCST` stands for concrete syntax tree (as opposed to abstract syntax tree).

Reply:
Didn't think this one through. Thanks.